### ISSUE TITLE
Can I use this package "extended-google-translate-api" so as to translate free and unlimited pages with google translate?

### ISSUE THREAD

**Issue by ioanf2** [FreddieDeWitt/google-translate-extended-api#9]
hello. Can I use this package "extended-google-translate-api" so as to translate free and unlimited pages with google translate? 

The question is, how good and accurate is the translation?

**Comment by FreddieDeWitt** [comment]
Yes, you can. As the module uses google translate data, the translations are as good (or as bad) as through google translate.

However, there will be problems when translating big chunks - unfortunately, such unofficial API only supports text until 5000 symbols (give or take) and your pages are probably bigger than that, so you would have to split you pages into chunks and translate them one by one (please note that you should also split by sentences, otherwise it won't work well).

Please take a look at [this issue](https://github.com/vitalets/google-translate-api/issues/20) for [this translation](https://www.npmjs.com/package/@vitalets/google-translate-api) module, it contains more information and some examples on how to implement what you need.

Please do ask if you have other questions.
